WooCommerce/Classes/ViewRelated/Dashboard/Blaze/UserDefaults+Blaze.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+import Combine
```

Comment:
nit: Combine is not used.

Reply:
Removed in 7c2c66d12939d9c7517ff104c58bc661cd4100f4.